docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -708,11 +708,19 @@ Backend     SQL Equivalent
 ==========  ======================================
 PostGIS     ``ST_DWithin(poly, geom, 5)``
 Oracle      ``SDO_WITHIN_DISTANCE(poly, geom, 5)``
+SpatiaLite  ``PtDistWithin(poly, geom, 5)``
 ==========  ======================================
 
 .. note::
+    The default units being queried against will vary by backend. On PostGIS
```

Comment:
I think the description above handles what is going on already. This second note isn't necessary